Description: Add missing <memory> include for std::unique_ptr in tips_agent.h
 The tips_agent.h header uses std::unique_ptr but doesn't include <memory>.
 With system libc++ on Debian, the header is not transitively included.
Author: Juan Manuel Méndez Rey <vejeta@gmail.com>
Bug-Debian: https://bugs.debian.org/915400
Forwarded: not-needed
Last-Update: 2025-12-04

diff --git a/chrome/browser/notifications/scheduler/public/tips_agent.h b/chrome/browser/notifications/scheduler/public/tips_agent.h
index 4dc5c23..2ada7f1 100644
--- a/chrome/browser/notifications/scheduler/public/tips_agent.h
+++ b/chrome/browser/notifications/scheduler/public/tips_agent.h
@@ -5,6 +5,8 @@
 #ifndef CHROME_BROWSER_NOTIFICATIONS_SCHEDULER_PUBLIC_TIPS_AGENT_H_
 #define CHROME_BROWSER_NOTIFICATIONS_SCHEDULER_PUBLIC_TIPS_AGENT_H_

+#include <memory>
+
 #include "chrome/browser/notifications/scheduler/public/notification_scheduler_types.h"

 namespace notifications {
